app/dfda/components/StudyCard.tsx

```
@@ -0,0 +1,168 @@
+        {study.studyText?.participantInstructions && (
+          <div className="neobrutalist-container bg-yellow-50 p-4 mb-6">
+            <h3 className="neobrutalist-h3 mb-2">How to Participate</h3>
+            <SanitizedContent html={study.studyText.participantInstructions} />
+          </div>
+        )}
+      </div>
+
+      {/* Statistics */}
+      {study.statistics && <StudyStatistics statistics={study.statistics} />}
+
+      {study.causeVariable && study.causeVariable.charts && (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)